mcf-browser/src/back-end/server-api.js

```diff
@@ -54,11 +54,15 @@ async function readFileList(fileList) {
             const mcfParser = new ParseMcf(tmcf.name + '&' + file.name);
             return mcfParser.parseMcfStr(mcf);
           });
-        finalReturn['errMsgs'] =
+
+        if(tmcfOut['errMsgs'] && tmcfOut['errMsgs'].length !== 0) {
+
+          finalReturn['errMsgs'] =
           finalReturn['errMsgs'].concat({
```

Comment:
The indents seem off, could you run lint?

Reply:
Is there a specific linter we use in DC?

Reply:
Website uses this - https://github.com/datacommonsorg/website/blob/master/run_test.sh#L49-L52